Makefile

```diff
@@ -146,9 +146,9 @@ images/test/bases:  ## Download base images for custom test images
 
 # Build a single container: <image name> <containerfile>
 define do-build-container-image =
-@ echo "$(ccgreen)Building$(ccend) $(ccred)${1}$(ccend) $(ccgreen)image...$(ccend)"
+@ echo "$(ccred)$$(date '+%x %T')$(ccend) $(ccgreen)Building$(ccend) $(ccred)${1}$(ccend) $(ccgreen)image...$(ccend)"
```

Comment:
Could we use date in the `YYY-MM-DD` format? I get this in my locale:

```
6.3.2025 13:15:34 Building tmt/container/test/alpine:latest image...
```

Seems a bit weird.

Reply:
> Could we use date in the YYY-MM-DD format?

Yes.

> Seems a bit weird.

Yes. Blame your locale of choice ;)

Reply:
Changed in c988014f